manifests/plugin/apache.pp

```diff
@@ -1,17 +1,23 @@
 # https://collectd.org/wiki/index.php/Plugin:Apache
 class collectd::plugin::apache (
-  $ensure         = present,
-  $manage_package = $true,
-  $instances      = { 'localhost' => { 'url' => 'http://localhost/mod_status?auto' } },
-  $interval       = undef,
+  $ensure                  = present,
+  $manage_package          = $true,
+  $instances               = { 'localhost' => { 'url' => 'http://localhost/mod_status?auto' } },
+  $interval                = undef,
+  $package_install_options = $collectd::params::package_install_options,
```

Comment:
My one question here is: what's our interface? Should we use `$collectd::package_install_options` for this? What would the user expect here? I'm not sure, that's why I'm posing the question.